core/frontend/src/libs/MAVLink2Rest/index.ts

```diff
@@ -1,11 +1,17 @@
 /* eslint-disable  @typescript-eslint/no-explicit-any */
 // The library is an interface for MAVLink objects, messages can by of any type
 
+import axios from 'axios'
+
+import { Dictionary } from '@/types/common'
+
 import Endpoint from './Endpoint'
 import Listener from './Listener'
 
-interface Dictionary<T> {
-  [key: string]: T;
+// Maps message names to message IDs
+// TODO: should we get this generated somehow?
```

Comment:
I would rather not have mavlink as a submodule in this project, if possible, could mavlink2rest provide this info, @patrickelectric ? I would even manually curate this list rather than include pymavlink...